Library/Tracking/KSRAnalytics.swift

```diff
@@ -633,6 +624,29 @@ public final class KSRAnalytics {
     }
   }
 
+  public func trackSearchTabBarClicked(prevTabBarItemLabel: TabBarItemLabel) {
+    let pageContext: PageContext
+    switch prevTabBarItemLabel {
+    case .activity:
+      pageContext = .activities
+    case .discovery:
+      pageContext = .discovery
+    case .profile:
+      pageContext = .profile
+    default:
+      pageContext = .search
```

Comment:
Dashboard is currently not covered because we don't have it defined yet. Laurie specifically mentioned it in the ticket.

`but creator_dashboard and messages are both screens we haven’t worked with yet so it’s ok to skip those if that’s easier.`

Reply:
I still think sending `.search` is bad practice. We should probably just return or break on that case so we aren't sending "noisy" data.

Reply:
So reason is that Laurie once mentioned that not sending `page_context` interfere with the analytics.

Reply:
I understand. I don't think this is a good idea for us, specifically but it is a Product requirement so we will leave it as is.